src/Magnum/LibOvrIntegration/Hmd.h

```diff
@@ -0,0 +1,233 @@
+#ifndef Magnum_LibOvrIntegration_Hmd_h
+#define Magnum_LibOvrIntegration_Hmd_h
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2015
+              Jonathan Hale <squareys@googlemail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Class @ref Magnum::LibOvrIntegration::Hmd and class @ref Magnum::LibOvrIntegration::SwapTextureSet.
+ */
+
+#include <bits/unique_ptr.h>
+#include <OVR/OVR_CAPI.h>
+#include <Magnum/Texture.h>
+#include <Magnum/Magnum.h>
+
+#include "visibility.h"
+#include "Conversion.h"
+#include "LibOvrIntegration.h"
+
+namespace Magnum { namespace LibOvrIntegration {
+
+/**
+@brief SwapTextureSet
+
+Contains an array of texture which can be rendered to an hmd display by the
+libOVR compositor after applying distortion to it.
+
+@todoc
+
+@see @ref Hmd
+*/
+class MAGNUM_LibOvrIntegration_EXPORT SwapTextureSet {
+public:
+    /**
+     * @brief Constructor.
+     * @param hmd Hmd for which this SwapTextureSet is created.
+     * @param format Texture format.
+     * @param size Size for the textures.
+     */
+    explicit SwapTextureSet(const Hmd& hmd, TextureFormat format, Vector2i size);
+    ~SwapTextureSet();
+
+    /** @brief Currently active texture in the set. */
+    Texture2D& getActiveTexture() const;
+
+    /**
+     * @brief Increment to use the next texture in the set.
+     * @return Reference to self (for method chaining)
+     */
+    SwapTextureSet& increment() {
+        _swapTextureSet->CurrentIndex = (_swapTextureSet->CurrentIndex + 1) % _swapTextureSet->TextureCount;
+        return *this;
+    }
+
+    /** @brief The underlying ovrSwapTextureSet. */
+    ovrSwapTextureSet* getOvrSwapTextureSet() const {
```

Comment:
There wasn't, I just need to keep the `_swapTextureSet` as a pointer, because `ovrHmd_CreateSwapTextureSetGL()` expects `ovrSwapTextureSet**`.